packages/ui-map-components/src/components/TilePicker/TileControl.tsx

```diff
@@ -18,8 +18,7 @@ const StyledButton = styled(Button)<{
   display: block;
   pointer-events: auto;
   box-shadow: none;
-  background: ${({ theme }) =>
-    theme.palette.type === 'light' ? 'rgba(255, 255, 255, 0.9)' : 'rgba(43, 45, 56, 0.9)'};
+  background: ${({theme}) => theme.navigationBtn.main};
```

Comment:
So we can't use non-mui theme variables (e.g. custom ones like this) in the ui-components (and similar) libraries, because not all the packages that consume this will have this theme structure. So perhaps you can just hardcode it in after all